JuliaPackaging/CMakeWrapper.jl
# GitHub Actions

- Fixing `compat` bounds which caused windows to fail to build
- Removed Manifest
- Using GitHub actions instead of Travis and Appveyor
- Testing build from source


Fixes https://github.com/JuliaPackaging/CMakeWrapper.jl/issues/15
Fixes https://github.com/JuliaPackaging/CMakeWrapper.jl/issues/14
Fixes https://github.com/JuliaPackaging/CMakeWrapper.jl/issues/22

To make CodeCov working, you should add the `CODECOV_TOKEN` to the secrets:
- Copy your token from here  https://codecov.io/gh/JuliaPackaging/CMakeWrapper.jl/settings
- Add it here with the name `CODECOV_TOKEN`: https://github.com/JuliaPackaging/CMakeWrapper.jl/settings/secrets


**Notice**: The Appveyor app should be removed from the repository, otherwise it will throw errors because of missing config: 
https://github.com/JuliaPackaging/CMakeWrapper.jl/settings/installations

## Ready
